src/storage/src/healthcheck.rs

```diff
@@ -642,22 +456,26 @@ where
                                   {last_reported_status:?} -> {:?}",
                             Some(&new_status)
                         );
-                        if let Some((status_shard, persist_client)) = persist_details {
-                            health_operator_impl
-                                .record_new_status(
-                                    *id,
-                                    new_status.name(),
-                                    new_status.error(),
-                                    new_status.hints().unwrap_or(&BTreeSet::new()),
-                                    new_status.errors().unwrap_or(&BTreeMap::new()),
-                                    now.clone(),
-                                    persist_client,
-                                    *status_shard,
-                                    schema,
-                                    write_namespaced_map,
-                                )
-                                .await;
+
+                        let mut timestamp = mz_ore::now::to_datetime(now());
+
+                        // Ensure that timestamps for updates are monotonically increasing
+                        if let Some(last_ts) = last_status_occurred_at {
+                            if *last_ts == timestamp {
+                                timestamp = timestamp + chrono::Duration::milliseconds(1);
```

Comment:
Do we have a guarantee that `now()` is monotonic? If not this could be done more defensibly like so

```suggestion
                            if timestamp <= *last_ts {
                                timestamp = *last_ts + chrono::Duration::milliseconds(1);
```

However I'm a bit skeptical of the need to do this kind of thing here or the need for monotonic time at all. This is just a log like the logs we put in stdout. Why not just use `now()`?

Reply:
honestly you are right, this is not really required; its from the old rebase i did